code/mob/melee_attack_procs.dm

```diff
@@ -995,7 +995,7 @@
 			target.TakeDamage(def_zone, (damage_type != DAMAGE_BURN ? damage : 0), (damage_type == DAMAGE_BURN ? damage : 0), 0, damage_type)
 
 			if ((damage_type & (DAMAGE_CUT | DAMAGE_STAB)) || bleed_always)
-				take_bleeding_damage(target, owner, damage + bleed_bonus, damage_type)
+				take_bleeding_damage(target, owner, damage + bleed_bonus, damage_type, is_crit=stamina_crit)
```

Comment:
Somewhat apprehensive about coupling stamina crits to bleeding effects, especially considering one of the weapons already best at causing bleeding, chainsaws, have one of the highest crit chances.

Reply:
its definitely worth thinking of. buuut IMO stamcrits are a little fuzzy on what they can and can't do.

my take, longterm, is that  it's positive to move them towards being just 'crits' with appropriate effects based on the weapon type. stamina for blunt, bleed/damage for stabbing weapons. why would 5 STA damage knives ever need *stam*crits?

ultimately bleeding is still only capped at 5/t. the only reason it needs to scale so hard with high damage weapons is because youll be dead in 3/4 hits anyway (way before the bleed can do anything). IMO it helps give lasting impression to big weapons now none of them just stun you forever, since you need more than just a lil' styptic.